lib/notification-issue.coffee

```diff
@@ -190,6 +191,9 @@ class NotificationIssue
           return packName unless /^\.\./.test(relativePath)
       @getPackageNameFromFilePath(filePath)
 
+    packageName = /Failed to (load|activate) (the|a) (.*) package/.exec(message)?[3]
```

Comment:
I wonder if we should pass in the `packageName` as metadata to the notification instead of having to regex the message.